holoviews/streams.py

```diff
@@ -124,6 +129,8 @@ def __init__(self, preprocessors=[], source=None, subscribers=[], **params):
         self.subscribers = subscribers
         self.preprocessors = preprocessors
         self._hidden_subscribers = []
+        self._metadata = None
```

Comment:
Should probably be an empty dict and I would add a comment what it is for - right now it is about stopping loops.